packages/thirdweb/src/wallets/utils/generateWallet.ts

```diff
@@ -0,0 +1,28 @@
+import { secp256k1 } from "@noble/curves/secp256k1";
+import type { Account } from "../interfaces/wallet.js";
+import { toHex } from "../../utils/encoding/hex.js";
+import { privateKeyAccount } from "../private-key.js";
+import type { ThirdwebClient } from "../../client/client.js";
+
+export type GenerateWalletOptions = {
+  client: ThirdwebClient;
+};
+
+/**
+ * Generates a new wallet with a random private key.
+ * @param options - The options for generating the wallet.
+ * @param options.client - The Thirdweb client to use for the generated wallet.
+ * @returns A Thirdweb wallet.
+ * @example
+ * ```ts
+ * import { generateWallet } from "thirdweb/wallets";
+ * const wallet = await generateWallet({ client });
+ * ```
+ * @walletUtils
+ */
+export async function generateWallet(
```

Comment:
since this generates an account lets call it `generateAccount` instead

actually thinking a bit further maybe more explicit `generateRandomAccount`

between the 2 I don't have a strong preference, just not `wallet` since it's not one :)